packages/nimble-components/src/rich-text/editor/tests/rich-text-editor.stories.ts

```diff
@@ -108,6 +108,9 @@ const metadata: Meta<RichTextEditorArgs> = {
             <${mappingUserTag} key="${x => mentionDataSets[x.mentionData].href}3" display-name="Sue Ann"></${mappingUserTag}>
```

Comment:
When you open the listbox in storybook it is clipped by the bounds of its container. We have a pattern to fix this which you can see in [these other stories](https://github.com/search?q=repo%3Ani/nimble%20disableStorybookZoomTransform&type=code)

![image](https://github.com/ni/nimble/assets/10500124/81cc7e3e-8658-43d9-85fd-18a281aef585)

![image](https://github.com/ni/nimble/assets/10500124/4f3551a2-6867-4256-a63e-b4f5418998d2)

Reply:
Updated